lib/posthog/send_feature_flags_options.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module PostHog
+  # Options for configuring feature flag behavior in capture calls
+  class SendFeatureFlagsOptions
+    attr_reader :only_evaluate_locally, :person_properties, :group_properties
```

Comment:
I see `only_evaluate_locally` defined here, but I don't see the code where we consume this attribute.